packages/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/utils/DependencyUtils.kt

```diff
@@ -69,6 +69,14 @@ internal object DependencyUtils {
               .using(it.module("${groupString}:hermes-android:${versionString}"))
               .because(
                   "The hermes-engine artifact was deprecated in favor of hermes-android due to https://github.com/facebook/react-native/issues/35210.")
+          if (groupString != DEFAULT_GROUP_STRING) {
```

Comment:
Can you please update this test (or better create a new one):
`configureDependencies_withVersionStringAndGroupString_appliesOnAllProjects`

Specifically you should assert that `com.facebook.react:react-android` and `com.facebook.react:hermes-android` are not in the forced modules

Reply:
This will need a different test, as this change does not appear to have any effect on the forced modules.